articles/templates/articles/blocks/image_block.html

```diff
@@ -0,0 +1,5 @@
+{% load wagtailimages_tags %}
```

Comment:
This is just a placeholder template.  It should be updated for the theming (OCW-86)